OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -172,6 +171,17 @@ public void onReceivedHttpError(WebView view, WebResourceRequest request, WebRes
         }
     }
 
+    /**
+     * Method is usable for Android 6.0 and below otherwise app didn't get the control when a URL is
+     * about to be loaded in the current WebView.
+     * Ref: https://developer.android.com/reference/android/webkit/WebViewClient#shouldOverrideUrlLoading(android.webkit.WebView,%20java.lang.String)
+     */
```

Comment:
IMO: following this better.

```java
     /**
     * Method is used for Android 6.0 and below otherwise app didn't get the control when a URL is
     * about to be loaded in the current WebView.
     * Ref: https://developer.android.com/reference/android/webkit/WebViewClient#shouldOverrideUrlLoading(android.webkit.WebView,%20java.lang.String)
     */
```

Reply:
still needs to update.